### ISSUE TITLE
HW3 Grades uploaded

### ISSUE THREAD

**Issue by jsjason** [swsnu/bd2017#22]
안녕하세요, TA 정주성입니다.

숙제3 채점이 완료되어 eTL 개인 성적표 화면에서 성적을 확인하실 수 있습니다.
숙제3의 만점은 10점입니다.
eTL에서는 성적이 백분율(%)로 표기되는 것 같으니, 10점 만점에 본인의 백분율을 곱하시면 본인의 성적을 정확히 알 수 있습니다 (예로 87.6%는 8.76점으로 환산됩니다).

점수에 관해 문의가 있으신 분들은 조교 메일 bd-ta@spl.snu.ac.kr 로 이메일 주시면 되겠습니다.

많은 분들이 궁금해 하실 공통적인 사항들에 대해서는 아래에 부연 설명을 드리겠습니다.
- Parameter Tuning: Batch size, learning rate와 같은 training parameter를 수정할 시 training에 어떤 영향을 미치는지를 분석한 결과를 레포트에 적어주실 것을 기대했으나, 레포트에 해당 부분에 관한 설명이나 분석을 넣으지 않으신 분들이 많았습니다. 최소한의 분석 결과를 기재해주신 분들께만 점수를 드렸습니다. 단순히 '이러이러한 parameter를 사용해서 training시켰다'의 설명만으로는, parameter tuning 분석을 하셨는지 알 수 없었기 때문에 점수를 드리지 않았습니다.
- Custom Model: Reference 논문의 VGG-M 모델 외에, 하나의 모델을 추가로 정하여 (custom이든, 다른 유명한 모델이든) training 및 inference를 진행하는 custom model 분석을 하지 않으신 분들께는 해당 부분에서 점수를 드리지 않았습니다.
- Test Accuracy: 숙제 문서에 명시한 25%의 test accuracy는, (최적화 없는) 정확한 모델을 적당한 parameter로 gpu위에서 training 시켰을 때, 짧은 시간 만에 얻을 수 있는 수치입니다. 모델을 정확히 구현하셨더라도 parameter 조정에서 실수하셔서 해당 test accuracy를 얻지 못하신 분들께는 test accuracy 점수를 드리지 않았습니다 (모델 구현 점수와 test accuracy 점수는 별개입니다).

감사합니다.
정주성 드림
